ui/shared/components/panel/search/constants.js

```diff
@@ -375,6 +375,114 @@ export const LOCATION_FIELDS = [
   },
 ]
 
+export const IN_SILICO_FIELDS = [
+  {
+    name: 'cadd',
+    label: 'CADD',
+    labelHelp: 'Enter a numeric value for Cadd prediction. During filtering, some variants might be left out, since their annotation value might be rounded up or down from their original value',
+    component: StyledSemanticInput,
+    inputType: 'Input',
+  },
+  {
+    name: 'revel',
+    label: 'Revel',
+    labelHelp: 'Enter a numeric value for Revel prediction. During filtering, some variants might be left out, since their annotation value might be rounded up or down from their original value',
+    component: StyledSemanticInput,
+    inputType: 'Input',
+  },
+  {
+    name: 'primate_ai',
+    label: 'Primate AI',
+    labelHelp: 'Enter a numeric value for Primate AI prediction. During filtering, some variants might be left out, since their annotation value might be rounded up or down from their original value',
+    component: StyledSemanticInput,
+    inputType: 'Input',
+  },
+  {
+    name: 'mpc',
+    label: 'MPC',
+    labelHelp: 'Enter a numeric value for MPC prediction. During filtering, some variants might be left out, since their annotation value might be rounded up or down from their original value',
+    component: StyledSemanticInput,
+    inputType: 'Input',
+  },
+  {
+    name: 'splice_ai',
+    label: 'Splice AI',
+    labelHelp: 'Enter a numeric value for Splice AI prediction. During filtering, some variants might be left out, since their annotation value might be rounded up or down from their original value',
+    component: StyledSemanticInput,
+    inputType: 'Input',
+  },
+  {
+    name: 'eigen',
+    label: 'Eigen',
+    labelHelp: 'Enter a numeric value for Eigen prediction. During filtering, some variants might be left out, since their annotation value might be rounded up or down from their original value',
+    component: StyledSemanticInput,
+    inputType: 'Input',
+  },
+  {
+    name: 'dann',
+    label: 'Dann',
+    labelHelp: 'Enter a numeric value for Dann prediction. During filtering, some variants might be left out, since their annotation value might be rounded up or down from their original value',
+    component: StyledSemanticInput,
+    inputType: 'Input',
+  },
+  {
+    name: 'strvctvre',
+    label: 'STRVCTVRE',
+    labelHelp: 'Enter a numeric value for STRVCTVRE prediction. During filtering, some variants might be left out, since their annotation value might be rounded up or down from their original value',
+    component: StyledSemanticInput,
+    inputType: 'Input',
+  },
+  {
+    name: 'polyphen',
+    label: 'Polyphen',
+    labelHelp: 'Enter a value for Polyphen prediction',
+    component: StyledSemanticInput,
+    inputType: 'Input',
```

Comment:
All predictor fields that are string values need to be dropdowns where users select one of the allowed options. Free text is completely unacceptable